src/FakeItEasy.Extensions.ValueTask/FakeItEasy.Extensions.ValueTask.csproj

```diff
@@ -1,8 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45;netstandard1.6;netstandard2.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>$(ValueTaskExtensionsTargetFrameworks)</TargetFrameworks>
     <SignAssembly>true</SignAssembly>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <BeforePack>AddNetStd21Placeholder;$(BeforePackOn)</BeforePack>

```

Comment:
Cleaner than depending on `_WalkEachTargetPerFramework`, which is a private target